maps/GoldenrodFlowerShop.asm

```diff
@@ -49,8 +47,6 @@ FlowerShopTeacherScript:
 FlowerShopFloriaScript:
 	faceplayer
 	opentext
-	checkevent EVENT_FOUGHT_SUDOWOODO
-	iftrue .FoughtSudowoodo
```

Comment:
I know for a fact that dabomstew will be locking the sudowoodo related skipping stuff behind a flag (cinco bingo players may not want it turned on). Consequently, we shouldn't delete things associated with this since its likely this check will get added back in later on with additional flag related checking.

Reply:
okay, so we can either
1. not delete this code, or
2. implement the flag.

Reply:
deleting this doesn't make sense if stew's gonna touch it himself later. just let him do whatever with it.